tests/kani/ForeignItems/lib.c

```diff
@@ -48,7 +55,11 @@ struct Foo2 {
 
 uint32_t S = 12;
 
-void update_static() { S++; }
+struct Unit update_static()
```

Comment:
This is something that ought to work once C-FFI is in place.  We should have comments and a tracking ticket explaining that this is a workaround we want to remove